packages/mui-joy/src/ToggleButtonGroup/ToggleButtonGroupContext.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+
+interface IToggleButtonGroupContext {
```

Comment:
Let's not prefix interfaces with `I`:
```suggestion
interface ToggleButtonGroupContextType {
```